pkg/options/websocket.yaml

```diff
@@ -0,0 +1,19 @@
+imports:
+  - time
+
+docs:
+  title: Websocket server
+
+props:
```

Comment:
Maybe we should add `logEnabled` (same as auth has it) to explicitly control logging in ws?